src/tools/dev/scripts/bv_support/bv_vtk.sh

```diff
@@ -74,11 +90,22 @@ function bv_vtk_depends_on
 function bv_vtk_info
 {
     info "bv_vtk_info"
-    export VTK_FILE=${VTK_FILE:-"VTK-9.2.6.tar.gz"}
-    export VTK_VERSION=${VTK_VERSION:-"9.2.6"}
-    export VTK_SHORT_VERSION=${VTK_SHORT_VERSION:-"9.2"}
-    export VTK_SHA256_CHECKSUM="06fc8d49c4e56f498c40fcb38a563ed8d4ec31358d0101e8988f0bb4d539dd12"
+
+    if [[ "$DO_VTK94" == "yes" ]] ; then
+        info "setting up vtk for version 9.4.1"
+        export VTK_VERSION=${VTK_VERSION:-"9.4.1"}
+        export VTK_SHORT_VERSION=${VTK_SHORT_VERSION:-"9.4"}
+        export VTK_SHA256_CHECKSUM=""
```

Comment:
no sha256sum?

Reply:
Thanks for the reminder. I also need to put the tarball in our repo.

Reply:
Done.